OrbitLinuxTracing/include/OrbitLinuxTracing/Tracer.h

```diff
@@ -88,9 +88,7 @@ class Tracer {
 
   static void Run(pid_t pid, uint64_t sampling_period_ns,
                   const std::vector<Function>& instrumented_functions,
-                  TracerListener* listener, bool trace_context_switches,
-                  bool trace_callstacks, bool trace_instrumented_functions,
-                  bool trace_gpu_driver,
+                  TracerListener* listener, TracingOptions tracing_options,
```

Comment:
Not that it makes a different, but you can pass `tracing_options` as const-ref.

Reply:
Sure, I'll do so